tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
+    - items to evaluate technological and architectural health of projects
+- Community outreach on any items within scope, for instance, social media/conferences/podcasts/meetups et cetera
+- Collaboration with other sustainability efforts
+
+### Out of scope
+- Form an umbrella organization beyond the CNCF
+- Establish a compliance and standards body beyond the CNCF space
+- Evaluate individual company infrastructures
+- Focus outside of Cloud Native Technologies, according to the [CNCF Cloud Native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md)
+
+## Deliverables to ToC
+- landscape for the carbon and energy metrics/measurements/management savings
+- recommendations for new CNCF projects where there are clear use cases or where exploration is needed
+- regular reporting to ToC on ongoing and completed work
+- recommend projects for sandbox and graduation status
+- suggestions for improvements for CNCF internal processes, for example, education for sustainability
+
+## Audiences
+
+- Education - audience is end users, developers, stakeholders
+- Project intelligence - audience is TOC / TOC Community
+- External collaboration - organizations with the same mindset
+
+#### Operations
+
+Environmental Sustainability TAG operations are consistent with standard TAG operating guidelines provided by the [CNCF Technical Oversight Committee TOC](https://github.com/cncf/toc).
+
+## Acknowledgement
+The format of this proposal is inspired from [TAG security](https://github.com/cncf/tag-security/blob/main/governance/charter.md#operations).
+The initial doc was taken from the [Environmental WG charter](https://docs.google.com/document/d/1JaF7lSUmLQ2zmScmca6UF7PgbjMzSxjhhjx2LThThaY/edit#heading=h.x77xxicolnig) and modified by the community.
+
+## Contributors
```

Comment:
please move this to the git commit message. It does not belong here.

Reply:
Agreed with @dims

Reply:
Adding to the other PR.

Reply:
Removed and added in commits.  Thank you.